NEWS

```diff
@@ -1,3 +1,100 @@
+WiredTiger release 2.6.1, 2015-05-13
+------------------------------------
+
+The WiredTiger 2.6.1 release contains new features, minor API changes and bug
+fixes.
+
+New features:
+
+* Add the ability to disable write ahead logging at a per-table granularity.
+  Accessed via log=(enabled) configuration for WT_SESSION::create API.
+  Partial logging has serious implications for recovery, it should be used
+  with caution.
+  refs #1989
+
+* Add the ability to asynchronously flush transactions via a new
+  WT_SESSION::transaction_sync API.
+  refs WT-1908, #1943
+
+* Add the ability to create a named in-memory snapshot via a new
+  WT_SESSION::snapshot API.
+  refs WT-1839
+
+* Move the sync configuration setting from WT_SESSION::begin_transaction to
+  WT_SESSION::commit_transaction. Some applications don't know whether they
+  need to sync a transaction to disk until they come to commit.
```

Comment:
Not only did the `sync` configuration setting move, but it changed.  The `background` setting is new to sync this transaction in the background.  The `true` or `false` has been changed to `on` or `off`.